app/views/users/show.html.erb

```diff
@@ -0,0 +1,57 @@
+<h1>Show User Details</h1>
+<h3 class="user-id">User ID: <%= @user.id %></h3>
+
+<main id="user-details" class="page">
+  <div id="user-page">
+    <div class="btn-container">
+      <div class="home-link-container ">
+      <%= link_to "Home", root_path, class: "home-link2" %>
+      </div>
+      <div class="back-link-container ">
+        <%= link_to "Back", :back, class: "back-link2" %>
+      </div> 
+    </div>
+  </div>
+  
+
+  
+  <div class="user-details-container ">
+    
+    <div class=" user-item">
+       <div class="user-image">
+ <%= image_tag('user.png', width: "150px", height: "100px") %>
+ </div>
+ <div class="user-details">
+      <h2 class="user-name"><%= @user.name %></h2>
+      <p class="user-posts-counter">Number of posts: <%= @user.posts_counter %></p>
+      </div>
+    </div>
```

Comment:
- [Optional] Hi @Kel-nana, let's kindly create modular codes by keeping our codebase DRY.  Using partials, we can enable the reuse of code across different parts of the application thus reducing redundancy and increasing efficiency. This same piece of codes is found in the index.html.erb file.

Reply:
implemented